generator/testdata/rust/openapi/golden/model.rs

```diff
@@ -10,7 +10,7 @@ use serde::{Deserialize, Serialize};
 pub struct ListLocationsResponse {
 
     /// A list of locations that matches the specified filter in the request.
-    pub locations: ,
+    pub locations: Vec<>,
```

Comment:
nit: this looks like it improved, but something is still missing. Well know cloud location.

Reply:
I think the merge with #103 will fix those too.